src/components/Modal/Modal.tsx

```diff
@@ -38,34 +44,67 @@ export const Modal = ({
   title,
   ...wrapperProps
 }: Props): ReactElement => {
+  const focusableElementSelectors =
+    'a[href]:not([tabindex="-1"]), button:not([disabled]), textarea:not([disabled]):not([tabindex="-1"]), input:not([disabled]):not([tabindex="-1"]), select:not([disabled]):not([tabindex="-1"]), area[href]:not([tabindex="-1"]), iframe:not([tabindex="-1"]), [tabindex]:not([tabindex="-1"]), [contentEditable=true]:not([tabindex="-1"])';
```

Comment:
Is this based on any documentation or stackoverflow post? would be good to comment with a link to that.

Reply:
@sowasred2012 You resolved this one, but I don't see any mention where does this selector come from?